src/System.ComponentModel/ref/System.ComponentModel.cs

```diff
@@ -9,7 +9,7 @@ namespace System
 {
     public partial interface IServiceProvider
     {
-        object GetService(System.Type serviceType);
+        object? GetService(System.Type? serviceType);
```

Comment:
Checked some implementation within corefx (`System.ComponentModel.MarshalByValueComponent and System.ComponentModel.Design.ServiceContainer.`) from `System.ComponentModel.TypeConverter`

Reply:
I think `serviceType` should be non-nullable.  Looking at a bunch of implementations in Visual Studio, many access `serviceType.GUID` without first validating it's not null.

Reply:
Good catch, in the implementation i was looking at was checking null but was not throwing or referencing it directly, so thought null will be safe